backend/tex/aux_parser.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+import json
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--file', help='filepath to input aux')
+args = parser.parse_args()
+
+f = open(args.file, 'r')
+PREFIX = '\\newlabel'
+
+listOfLabels = []
+
+def parseArg(s, i):
+    end = s.find("}", i + 1)
+    if end == -1:
+        return None, len(s)
+    return s[i + 1: end], end + 1
+
+def parseLabel(s, prefix = PREFIX):
+    s = s.replace(prefix, '')
+
+    ind = 0
+    args = []
+    while ind < len(s):
+            if s[ind] == '{' and s[ind+1] == '{':
+                ind += 1
+            arg, ind = parseArg(s, ind)
+            if arg != None:
```

Comment:
pls don't compare to `None` by `!=`, there is a `is not` operator for [that](http://stackoverflow.com/a/3965129)